arcgis-ios-sdk-samples/Route and directions/Display route layer/DisplayDirectionsViewController.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayDirectionsViewController: UITableViewController {
+    var directions = [String]()
+    
+    // MARK: Initializers
+    
+    init?(
+        coder: NSCoder,
+        directions: [String]
+    ) {
+        self.directions = directions
+        super.init(coder: coder)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - UITableViewDataSource
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return directions.count
```

Comment:
```swift
       directions.count
```
I tend to omit return and use the syntactic sugar, [ref](https://www.swiftbysundell.com/questions/when-can-the-return-keyword-be-omitted/) but it's totally a personal style choice so I won't make you make this change here.